test/unit-tests/common/test_image_star.py

```diff
@@ -4,71 +4,88 @@
     :license: BSD-2-Clause, see LICENSE for details.
 """
 
-from sphinxcontrib.confluencebuilder.state import ConfluenceState
-from sphinxcontrib_confluencebuilder_util import ConfluenceTestUtil as _
-from sphinxcontrib.confluencebuilder.builder import ConfluenceBuilder
 import os
 import re
-import unittest
-import tempfile
 import shutil
+import tempfile
+import unittest
+
+from sphinxcontrib.confluencebuilder.state import ConfluenceState
+from sphinxcontrib_confluencebuilder_util import ConfluenceTestUtil as _
+from sphinxcontrib.confluencebuilder.builder import ConfluenceBuilder
 
 
 class TestImageStar(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        cls.extension_pattern = '@REPLACE_EXTENSION@'
-        cls.test_dir = os.path.dirname(os.path.realpath(__file__))
-        cls.image_file = os.path.join(cls.test_dir, 'assets', 'image01.png')
+        test_dir = os.path.dirname(os.path.realpath(__file__))
+        cls.image_file = os.path.join(test_dir, 'assets', 'image01.png')
+        cls.image_basename = cls.file_basename(cls.image_file)
+        cls.datasetdir = tempfile.mkdtemp()
+
+    def tearDown(self):
```

Comment:
Correct! `tearDown` works too because it is called after every single test. But, as datasetdir is created in `setUpClass`, it makes more sense to clean it in `tearDownClass`.